packages/discovery-provider/src/api/v1/models/tracks.py

```diff
@@ -126,6 +127,47 @@
     },
 )
 
+
+tip_gate = ns.model("tip_gate", {"tip_user_id": fields.String(required=True)})
```

Comment:
should these models be outside of tracks? e.g. albums?

Reply:
Yeah would make sense for gating conditions to be in their own model file. ContentAccess?